### PR TITLE
fix(profiling): change default timeout to match tracer

### PR DESCRIPTION
**Pull request by jd**
This changes the default timeout for profiling agent connection to 2 seconds
rather than 10 seconds, matching the tracer's default.

### CODE REVIEW

**Review by majorgreys — DISMISSED** [review]

File: releasenotes/notes/profiling-default-timeout-agent-43b19237a21fd337.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+other:
+  - |
+    The default connection timeout for the profiling agent has now been reduced
+    from 10 to 2 seconds to match the tracer behaviour.
```
Comment (P403n1x87) [inline releasenotes/notes/profiling-default-timeout-agent-43b19237a21fd337.yaml:5]:
```suggestion
    from 10 to 2 seconds to match the tracer behavior.
```

**Review by P403n1x87 — CHANGES_REQUESTED** [review]
Please use AmE to make the spellchecker happy 🙂

File: releasenotes/notes/profiling-default-timeout-agent-43b19237a21fd337.yaml [hunk 2]
```diff
@@ -2,4 +2,4 @@
 other:
   - |
     The default connection timeout for the profiling agent has now been reduced
-    from 10 to 2 seconds to match the tracer behaviour.
+    from 10 to 2 seconds to match the tracer behavior.
```
Comment (Kyle-Verhoog) [inline releasenotes/notes/profiling-default-timeout-agent-43b19237a21fd337.yaml:5]: :cry: 🇨🇦 🇬🇧 🇪🇺

**Review by Kyle-Verhoog — APPROVED** [review]

File: releasenotes/notes/profiling-default-timeout-agent-43b19237a21fd337.yaml (hunk 2 above)
Comment (brettlangdon) [reply releasenotes/notes/profiling-default-timeout-agent-43b19237a21fd337.yaml:5]: Is it possible to support both?

Personally the localization differences don't really make much of a difference to me. Would be better to allow either ?

Comment (P403n1x87) [reply releasenotes/notes/profiling-default-timeout-agent-43b19237a21fd337.yaml:5]: It'd certainly increase the CI pass rate 🙂

Comment (jd) [reply releasenotes/notes/profiling-default-timeout-agent-43b19237a21fd337.yaml:5]: Je peux écrire en français sinon.

**Review by P403n1x87 — APPROVED** [review]

**Pull request merged**